test/unit/org/apache/cassandra/tools/BulkLoaderTest.java

```diff
@@ -36,9 +36,11 @@
     @Test
     public void testBulkLoader_NoArgs() throws Exception
     {
-        ToolRunner tool = runner.invokeClassAsTool("org.apache.cassandra.tools.BulkLoader");
-        assertEquals(1, tool.getExitCode());
-        assertTrue(!tool.getStderr().isEmpty());
+        try (ToolRunner tool = runner.invokeClassAsTool("org.apache.cassandra.tools.BulkLoader"))
+        {
+            assertEquals(1, tool.getExitCode());
+            assertTrue(!tool.getCleanedStderr().isEmpty());
```

Comment:
I know you are only refactoring the existing test, but this test verifies an error exists, but not that it is the correct error.  If BulkLoader changes then this test will only fail if bulk loader does something like prints a help without failing.  I thin we should validate that error contains `Missing sstable directory argument` as this is the actual expected error.

Reply:
+1000